pygls/server.py

```diff
@@ -197,6 +218,35 @@ def start_tcp(self, host, port):
         finally:
             self.shutdown()
 
+    def start_websocket(self, host, port, path='/'):
+        """Starts WebSocket server."""
+        logger.info('Starting WebSocket server on {}:{}'.format(host, port))
+
+        # TODO: Handle the path
+        self._stop_event = Event()
+        self.lsp._send_only_data = True  # Don't send headers within the payload
+
+        async def connection_made(websocket, cur_path):
```

Comment:
See above comment. I am not sure about the path argument, I need to take a look at websocket lib.